mozilla-release/toolkit/mozapps/installer/upload-files.mk

```diff
@@ -234,25 +236,28 @@ endif
 MAIN_RPM= $(MOZ_APP_NAME)-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU)$(RPM_PKG_SUFFIX)
 MAIN_DEB= $(MOZ_APP_NAME)-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU).deb
 # Create deb package
-RPM_CMD += && mv $(TARGET_CPU)/$(MAIN_RPM) $(_ABS_DIST)/
+RPM_CMD += && mv $(TARGET_CPU)/$(MAIN_RPM) $(ABS_DIST)/
 RPM_CMD += && fakeroot alien $(MAIN_RPM) --scripts
 # Rename newly created deb (alien does not provide such option)
-RPM_CMD += && mv *.deb $(MAIN_DEB)
+RPM_CMD += && mv `ls *.deb | head -1` $(MAIN_DEB)
 UPLOAD_EXTRA_FILES += $(MAIN_RPM)
-RPM_CMD += && mv $(TARGET_CPU)/$(MAIN_RPM) $(ABS_DIST)/
 UPLOAD_EXTRA_FILES += $(MAIN_DEB)
 
-#ifdef ENABLE_TESTS
-#TESTS_RPM=$(MOZ_APP_NAME)-tests-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU)$(PKG_SUFFIX)
-#UPLOAD_EXTRA_FILES += $(TESTS_RPM)
-#RPM_CMD += && mv $(TARGET_CPU)/$(TESTS_RPM) $(ABS_DIST)/
-#endif
-
-#ifdef INSTALL_SDK
-#SDK_RPM=$(MOZ_APP_NAME)-devel-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU)$(PKG_SUFFIX)
-#UPLOAD_EXTRA_FILES += $(SDK_RPM)
-#RPM_CMD += && mv $(TARGET_CPU)/$(SDK_RPM) $(ABS_DIST)/
-#endif
+ifeq (0,1)
+ifdef ENABLE_TESTS
+TESTS_RPM=$(MOZ_APP_NAME)-tests-$(MOZ_NUMERIC_APP_VERSION)-$(MOZ_RPM_RELEASE).$(BUILDID).$(TARGET_CPU)$(PKG_SUFFIX)
+UPLOAD_EXTRA_FILES += $(TESTS_RPM)
+RPM_CMD += && mv $(TARGET_CPU)/$(TESTS_RPM) $(ABS_DIST)/
+endif
+endif
+
+ifeq (0,1)
```

Comment:
@chrmod please add small comments with explanations on why you disable parts of code, when you do so.
After several merges even the blame history may be lost, making it hard to track back to the ticket ID.